packages/orchestrator-ui-components/src/graphqlQueries/subscriptionDetailQuery.ts

```diff
@@ -46,7 +46,10 @@ export const GET_SUBSCRIPTION_DETAIL_GRAPHQL_QUERY: TypedDocumentNode<
                     subscriptionInstanceId
                     inUseByRelations
                 }
-                processes(sortBy: { field: "startedAt", order: ASC }) {
+                processes(
+                    sortBy: { field: "startedAt", order: ASC }
+                    filterBy: { field: "isTask", value: "false" }
```

Comment:
Do we want hard coded values in our queries?

Reply:
For a payload query this looks fine to me.